benchmark/querystring/querystring-stringify.js

```diff
@@ -23,6 +26,16 @@ function main({ type, n }) {
       foo: 'bar',
       baz: 'quux',
       xyzzy: 'thu\u00AC'
+    },
+    array: {
+      foo: [],
+      baz: ['bar'],
+      xyzzy: ['bar', 'quux', 'thud']
+    },
+    array0: {
+      foo: [],
+      baz: [],
+      xyzzy: []
```

Comment:
These are edge cases that do not seem to be the typical use case. I would rather not add any further type that is not used in common cases.

Reply:
Thanks, removed the benchmark edge cases.